### PR TITLE
fix handling of ds3_str response types; rename (init_)ds3_get_objects…

### PR DESCRIPTION
**Pull request by DenverM80**
…_spectra_s3_request to (init_)ds3_get_objects_details_spectra_s3_request & update tests

### CODE REVIEW

File: src/ds3.c [hunk 1]
```diff
@@ -14784,9 +14784,10 @@ ds3_error* ds3_get_jobs_spectra_s3_request(const ds3_client* client, const ds3_r
 
     return _parse_top_level_ds3_job_list_response(client, request, response, xml_blob);
 }
-ds3_error* ds3_get_put_job_to_replicate_spectra_s3_request(const ds3_client* client, const ds3_request* request, ds3_str* response) {
+ds3_error* ds3_get_put_job_to_replicate_spectra_s3_request(const ds3_client* client, const ds3_request* request, ds3_str** response) {
```
Comment (rpmoore) [inline src/ds3.c]: Does 3.0 have replication?  I thought that was only with 3.2

Comment (DenverM80) [inline src/ds3.c]: its in the contract, apparently.  this is only intended for ds3 targets

File: test/search_tests.cpp [hunk 2]
```diff
@@ -159,7 +159,7 @@ BOOST_AUTO_TEST_CASE(get_objects_with_plus_query_param) {
 
 /* Disabling from nightly test until network timeout failure is resolved */
```
Comment (rpmoore) [inline test/search_tests.cpp]: We should create a jira task to track this.

Comment (DenverM80) [inline test/search_tests.cpp]: created https://jira.spectralogic.com/browse/CSDK-116

**Pull request merged**